tests/e2e/pact/specs/eps-sandbox.spec.ts

```diff
@@ -100,7 +100,7 @@ jestpact.pactWith(
           },
           willRespondWith: {
             headers: {
-              "Content-Type": "application/fhir+json; fhirVersion=4.0"
+              "Content-Type": "multipart/mixed; boundary=----=_MIME-Boundary"
```

Comment:
This expectation for the content-type feels like a bit of a mismatch, as the sandbox will be returning "Message Sent" instead of communicating with Spine, text/plain is probably more accurate

Reply:
Thats a bit of a special case for Sandbox mode though. Do you think we should return a mock spine response in that case?